layouts/default.vue

```diff
@@ -73,10 +74,7 @@ onMounted(() => {
           name="twitter:description"
           content="Investigating a strange world."
         />
-        <Meta
-          name="twitter:image"
-          content="https://media.wnyc.org/i/1200/600/l/80/2020/09/Radiolab_OG_Image_Options_3.png"
-        />
+        <Meta name="twitter:image" content="/images/default-og-image.png" />
```

Comment:
need to be content="/og-default-image.png" ... to match line 54?

Reply:
yes duh 🙃